src/scicookie/{{cookiecutter.project_slug}}/.gitignore

```diff
@@ -62,28 +68,69 @@ instance/
 # Scrapy stuff:
 .scrapy
 
+# Sphinx documentation
+docs/_build/
+
 # PyBuilder
+.pybuilder/
 target/
 
 # Jupyter Notebook
 .ipynb_checkpoints
 
+# IPython
+profile_default/
+ipython_config.py
+
 # pyenv
-.python-version
+#   For a library or package, you might want to ignore these files since the code is
+#   intended to run in multiple environments; otherwise, check them in:
+# .python-version
+
+# pipenv
+#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in version control.
+#   However, in case of collaboration, if having platform-specific dependencies or dependencies
+#   having no cross-platform support, pipenv may install dependencies that don't work, or not
+#   install all needed dependencies.
+#Pipfile.lock
+
+{%- if cookiecutter.build_system == "poetry" %}
```

Comment:
@Saransh-cpp, nice idea to have selectors for that!

I think we should add that files, but commented with `#`
because in theory, that files should be tracked by git in order to be reproducible

thoughts?

Reply:
Yes, keeping the files commented with `#` sounds good too. I think we should do that. Sorry for the extra work, @Naman-Priyadarshi!

Reply:
no worries!